main.go

```diff
@@ -127,11 +133,6 @@ func main() {
 		os.Exit(1)
 	}
 
-	inspect, _, err := dockerClient.ImageInspectWithRaw(context.Background(), originalImageID.String())
-	if len(inspect.RepoTags) > 0 {
-		return
-	}
```

Comment:
We still want this return here, otherwise it'll delete all the other image tags that exist.